services/web-server/test/session_store_test.js

```diff
@@ -119,13 +118,10 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
         store.touch('foo', { cookie: { maxAge: 300 } }, done);
       });
     });
-    test('it should error when touching the session', async function () {
+    test('it should not error when touching the session', async function () {
```

Comment:
perhaps this means I didn't read this review closely enough but why does this change from should-error to should-not-error?

Reply:
The reason I removed this was because we were using `session_add` in the `touch` method which will either create or update an existent entry. Now that I look closer, the touch method should be using `session_load` rather than `session_add`. I'll update this. Thanks for flagging this. This could have been a bug.